test/unit/specs/test_required_based_on_value.py

```diff
@@ -0,0 +1,25 @@
+"""
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: MIT-0
+"""
+import logging
+import test.fixtures.schemas
+from jsonschema import validate
+from test.testlib.testcase import BaseTestCase
+import cfnlint.helpers
+
+
+LOGGER = logging.getLogger('cfnlint.maintenance')
+LOGGER.addHandler(logging.NullHandler())
+
+
+class TestRequiredBasedOnValueSpec(BaseTestCase):
+    """Used for Testing Rules"""
+
+
+    def test_success_sbd_domain_removed(self):
+        """Success removal of SBD Domain form unsupported regions"""
```

Comment:
Some outdated copy-pasted code here
```suggestion


    def validate_additional_specs_schema(self):
```